index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE>
+<html>
+<head>
+     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Trial project</title>
+<link rel="stylesheet" href="styles.css"
```

Comment:
please, keep your codebase indentation to make more readable your code.

Reply:
- [ ] Good job on providing feedback here However, it may be helpful to provide specific examples of where the indentation can be improved to be more clear and easier to understand or provide resources on best practices for code formatting and organization. This will help the student to make the necessary changes and improve the overall readability and maintainability of their code easily.
- [ ] Also please take note that as part of the project requirements you have to write at least 2 comments in the review. So kindly add another comment.